templates/aggro_create.tmpl

```diff
@@ -39,6 +39,19 @@
                 <!-- consider https://github.com/williamtroup/Calendar.js/ -->
                 <input type="datetime-local" id="Start" name="Start" class="form-control">
 				<input type="hidden" id="TimeZone" name="TimeZone" >
+                <script type="text/javascript">
```

Comment:
never understood the convention of script in body vs head. Should probably kick this an clarity into a seperate file.

Reply:
Yeah, would love typescript and a bundler that code splits on routes and makes one js file per template/route, but I have no familiarity with gin and felt like overkill for this targeted improvement.